src/compiler/compile/compiler_errors.ts

```diff
@@ -261,5 +261,9 @@ export default {
 	cyclical_const_tags: (cycle: string[]) => ({
 		code: 'cyclical-const-tags',
 		message: `Cyclical dependency detected: ${cycle.join(' → ')}`
-	})
+	}),
+	invalid_style_directive: {
+		code: 'invalid-style-directive',
```

Comment:
Maybe this should be `invalid_component_style_directive` and `invalid-component-style-directive` since they are for components only.

Reply:
I've now renamed them accordingly.